pytest_tests/tests/object/test_object_api_bearer.py

```diff
@@ -17,6 +30,8 @@
 from neofs_testlib.shell import Shell
 from pytest import FixtureRequest
 
```

Comment:
Is this the way `black` formats the code?

Reply:
my IDE, make them live together